aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -345,6 +345,31 @@ void normal_kernel(Tensor& self, double mean, double std, c10::optional<Generato
   templates::cpu::normal_kernel(self, mean, std, generator);
 }
 
+
+static void rad2deg_kernel(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND2(kBFloat16, kHalf, iter.dtype(), "rad2deg_cpu", [&] {
+        cpu_kernel_vec(
+          iter,
+          [=](scalar_t a) -> scalar_t {
+            return static_cast<scalar_t>(180.0) * a / static_cast<scalar_t>(M_PI);
```

Comment:
Instead of performing this computation each time, would it make more sense to precompute 180/pi?